### ISSUE TITLE
The `gls` command is not accessible globally

### ISSUE THREAD

**Issue by sangalli406** [mauodias/gls#2]

**Comment by mauodias** [comment]
Do you mean running it directly, without specifying the path or running with `./`? To do this, the script needs to be in your `$PATH`, either by saving it directly in a folder that is contained in the env var, by creating an alias pointing to the script or by just creating a symlink (which I prefer).

Is this what you meant, @hscorrea ?

**Comment by sangalli406** [comment]
Basically, yes. Although, I think it would be nice to have this whole process documented or even automated.

What do you say, @mauodias ?

**Comment by mauodias** [comment]
Good point. A more detailed README will be added in the next commit, explaining how to 'install' the tool. A script may be considered in the future, but not immediately (I accept PRs, though!)
